docs/foundation/brand-colors.md

```diff
@@ -12,132 +12,287 @@ eleventyNavigation:
-        <div class="card border-0 mb-4">
-            <div class="py-20 bg-primary rounded-top"></div>
-            <div class="card-body">
-                <p class="mb-0 font-weight-bold">Primary</p>
-                <p class="mb-0"><code>#00284d</code></p>
-                <p class="mb-0"><code>$primary</code></p>
-            </div>
-        </div>
+Brand colors are the colors featured in an agency’s logo and other aspects of its visual identity. Pelican provides three default brand colors. These colors are based upon Louisiana’s official colors.
+
+<div class="row">
+  <div class="col-md-6 col-xl-4">
+    <div class="card border-0 mb-4">
+      <div class="py-20 bg-primary rounded-top"></div>
+      <div class="card-body">
+        <p class="mb-0 fw-bold">Primary</p>
+        <p class="mb-0">Hex Code: <code>#00284d</code></p>
+        <p class="mb-0">SCSS Var: <code>$primary</code></p>
+      </div>
     </div>
-    <div class="col-md-6 col-xl-3">
-        <div class="card border-0 mb-4">
-            <div class="py-20 bg-secondary rounded-top"></div>
-            <div class="card-body">
-                <p class="mb-0 font-weight-bold">Secondary</p>
-                <p class="mb-0"><code>#fe9903</code></p>
-                <p class="mb-0"><code>$secondary</code></p>
-            </div>
-        </div>
+  </div>
+  <div class="col-md-6 col-xl-4">
+    <div class="card border-0 mb-4">
+      <div class="py-20 bg-secondary rounded-top"></div>
+      <div class="card-body">
+        <p class="mb-0 fw-bold">Secondary</p>
+        <p class="mb-0">Hex Code: <code>#fe9903</code></p>
+        <p class="mb-0">SCSS Var: <code>$secondary</code></p>
+      </div>
     </div>
-    <div class="col-md-6 col-xl-3">
-        <div class="card border-0 mb-4">
-            <div class="py-20 bg-accent rounded-top"></div>
-            <div class="card-body">
-                <p class="mb-0 font-weight-bold">Accent</p>
-                <p class="mb-0"><code>#008282</code></p>
-                <p class="mb-0"><code>$accent</code></p>
-            </div>
-        </div>
+  </div>
+  <div class="col-md-6 col-xl-4">
+    <div class="card border-0 mb-4">
+      <div class="py-20 bg-accent rounded-top"></div>
+      <div class="card-body">
+        <p class="mb-0 fw-bold">Accent</p>
+        <p class="mb-0">Hex Code: <code>#008282</code></p>
+        <p class="mb-0">SCSS Var: <code>$accent</code></p>
+      </div>
     </div>
+  </div>
 </div>
 
 ## Using Brand Colors
 
-Some of the uses of each Brand Color are presented in these lists, but not limited to these items. If your digital product requires changing these colors, see [Agency Themes](/foundation/agency-theming/) for more information.
+Some for each Brand Color are presented in these lists, but not limited to only these items. These colros have been automatically assigned for you. If your digital product requires changing these colors, see [Agency Themes](/foundation/agency-theming/) for more information.
```

Comment:
"Some **uses** for each Brand Color are presented in these lists, but not limited to only these items. These **colors** have been automatically assigned for you."